docs/markdown/bar-series.md

```diff
@@ -117,3 +117,7 @@ The opacity for all elements in the series, this property will be over-ridden by
 #### stroke
 Type: `string|number`
 The outer color for all elements in the series, this property will be over-ridden by color specified in the data attribute.
+
+### style
+Type: `object`
+A list of CSS properties to style the series outside of the explicitly set properties. Note that it will override all other properties (ie fill, stroke, opacity, color).
```

Comment:
I'm not saying that we shouldn't repeat the props that are available to use, I said that I think we should not use the same property if it behaves differently. Passing multiples styles objects in a `style` property shouldn't be allowed